examples/image_classification/ImageClassificationExample.js

```diff
@@ -36,6 +39,8 @@ class ImageClassificationExample extends BaseCameraExample {
       case 'OpenCV.js':
         labelClasses = getTopClasses(output.tensor, output.labels, 3);
```

Comment:
@lionkunonly The case 'OpenVINO.js' uses the same code as case 'OpenCV.js', you could code as 

```JavaScript
      case 'OpenCV.js':
      case 'OpenVINO.js':
```

Reply:
Done